.github/pull_request_template.md

```diff
@@ -18,5 +18,5 @@ Please describe your pull request and link relevant issues.
 - [ ] Tests for the changes have been added
 - [ ] Necessary documentation is added (if appropriate)
 - [ ] Formatted with `cargo fmt --all`
-- [ ] Linted with `cargo clippy --all-features -- -D warnings`
-- [ ] Tested with `cargo test --workspace --all-features`
+- [ ] Linted with `cargo clippy --all-features --all-targets`
```

Comment:
Is there any particular reason why you remove the `-- -D warnings` part? Doing that will make it so that clippy warnings don't cause the pipeline to fail, is this what we want?

Reply:
The CI pipeline still denies warnings. It's just the PR template that I wanted to simplify.